module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,19 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+bool compare(const std::string& s1, const std::string& s2) {
+    return s1.size() < s2.size();
+}
+
+std::deque<std::string> lengthSort(std::forward_list<std::string>& str) {
+    str.sort();
+    std::deque<std::string> sortMe;
+
+    for (const auto& el : str) {
+        sortMe.push_back(el);
+    }
```

Comment:
You can initialize deque like this:
std::deque<std::string> rsortMe{str.begin(), str.end()};
There is no point for push_back for.